src/converse-register.js

```diff
@@ -241,6 +246,24 @@
                     return false;
                 },
 
+                chooseProvider: function () {
+                    /* Method called when registration_domain is provided
+                     * which loads the registration form directly
+                     */
+                    var form_help = document.getElementsByClassName('form-help');
```

Comment:
Use `querySelector('.form-help')` instead. Then it will return only the element, or `null` if no element is found.